CONTRIBUTING.md.template

```diff
@@ -1,7 +1,7 @@
-We love pull requests from everyone. Follow the thoughtbot [code of conduct]
-while contributing.
+We love pull requests from everyone. By participating in this project, you agree
+to abide by the thoughtbot [code of conduct].
 
-[code of conduct]: https://thoughtbot.github.com/code-of-conduct
+[code of conduct]: https://thoughtbot.com/code-of-conduct
```

Comment:
Is the URL on this going to be `/open-source-code-of-conduct` to distinguish from a later `/events-code-of-conduct`?